### ISSUE TITLE
Is possible concatenate from query ? How ?

### ISSUE THREAD

**Issue by jmencoder** [rakibtg/SleekDB#169]
I get the data in the next way

### My Example

$result = $tabla->select(["user","firstn","lastn","email","phones"])->OrderBy("desc")->fetch();

I need CONCAT **firstn** and **lastn**.

**Comment by Timu57** [comment]
Hi @jmencoder can you tell me a little bit more about your use case?

**Comment by Timu57** [comment]
Currently the only two ways I can think of are the following:

## Concat on insert

```sql
$store = new Store("user", __DIR__ . "/database");

$firstName = "Max";
$lastName = "Muster";

$newUser = [
  "firstname" => $firstName,
  "lastname" => $lastName,
  "fullname" => $firstName . " " . $lastName
]

$newUser = $store->insert($newUser);
```

## Concat on output

```sql
$store = new Store("user", __DIR__ . "/database");

$result = $store->orderBy("desc")->getQuery()->fetch();

echo "FirstName: " . $result["firstn"];

echo "LastName: " . $result["lastn"];

echo "FullName: " . $result["firstn"] . " " . $result["lastn"];
```

**Comment by jmencoder** [comment]
Thnks.

But, I need to do something like this.

SELECT **CONCAT(**_FIELD,_ '', _FIELD_**)** FROM TABLE.

**Comment by Timu57** [comment]
@jmencoder I am sorry but that is currently not possible. In #170 I describe adding that feature, so look forward for future releases :relaxed:

**Comment by Timu57** [comment]
Implemented with version 2.9
